index.js

```diff
@@ -76,28 +79,40 @@ if (['github', 'bitbucket'].indexOf(son.source) === -1) {
     process.exit(11);
 }
 
-// if a user is provided, we also need a password for the basic authentication
-if (user && !argv['password']) {
+//if json config file exists, use its credentials.
 
-    prompt.get(CONFIG.promptSchema, function(err, result) {
+if (fs.existsSync(configFilePath)) {
```

Comment:
Use [`is-there`](https://github.com/IonicaBizau/node-is-there) here since `exists*` will be deprecated.